mreg_cli/commands/host_submodules/rr.py

```diff
@@ -95,7 +95,7 @@ def hinfo_add(args: argparse.Namespace) -> None:
     host = host.refetch()
 
     if host.hinfo and host.hinfo.cpu == args.cpu and host.hinfo.os == args.os:
-        OutputManager().add_ok(f"Added HINFO record for {host.name.hostname}.")
+        OutputManager().add_ok(f"Added HINFO record for {host.name}.")
```

Comment:
```suggestion
        OutputManager().add_ok(f"Added HINFO record for {host.name!r}.")
```

Should probably be displayed in single quotes like most other places?

Reply:
Yes please.

Reply:
We should do a full pass and just do this.

Later. :)